Dockerfile

```diff
@@ -11,6 +11,8 @@ WORKDIR /app
 COPY package.json /app
 COPY yarn.lock /app
 RUN yarn install
+RUN apt-get update && apt-get install -y libnss3 libnspr4 libatk1.0-0 libatk-bridge2.0-0 libcups2 libdrm2 libxkbcommon0 libxcomposite1 libxdamage1 libxfixes3 libxrandr2 libgbm1 libasound2
```

Comment:
I got this error and used this answer to fix it but I'm not sure is it ok to do it. 
[https://stackoverflow.com/a/71128432](https://stackoverflow.com/a/71128432)

Reply:
This looks like an acceptable solution. I believe we had to do something similar for v1.